src/databricks/labs/ucx/hive_metastore/workflows.py

```diff
@@ -72,6 +72,9 @@ def migrate_views(self, ctx: RuntimeContext):
     def refresh_migration_status(self, ctx: RuntimeContext):
         """Refresh the migration status to present it in the dashboard."""
         ctx.tables_migrator.index_full_refresh()
```

Comment:
See other comment about moving this in `get_remaining_tables`

Reply:
Done